pypsa-to-iamc

```diff
@@ -0,0 +1,634 @@
+            
+              var[sh][sheet_var[sheet]+'|Electricity|Gases|Hydrogen'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='H2 Fuel Cell').filter(like =country)).sum(axis=1)))
+
+              #Energy generation : Oil 
+
+              var[sh][sheet_var[sheet]+'|Electricity|Liquids|Fossil'] = 0.001*ct*gen_des(time((n.links_t.p1.filter(like ='oil-').filter(like =country)).sum(axis=1)))
+
+              #Energy generation : Power to gas 
+              
+              var[sh][sheet_var[sheet]+'|P2G|Electricity'] = 0.001*ct*gen_des(time(n.links_t.p1.filter(like ='Sabatier').filter(like =country).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|P2G|Electricity']+= 0.001*ct*gen_des(time(n.links_t.p1.filter(like ='helmeth').filter(like =country).sum(axis=1))) 
+              var[sh][sheet_var[sheet]+'|P2G|Electricity']+= 0.001*ct*gen_des(time(n.links_t.p1.filter(like ='H2 Electrolysis').filter(like =country).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|P2G|Electricity']+= 0.001*ct*gen_des(time(n.links_t.p1.filter(like ='Fischer-Tropsch').filter(like =country).sum(axis=1)))               
+              
+            
+              #Energy generation : hydro (reservoir, ror)
+
+              var[sh][sheet_var[sheet]+'|Electricity|Hydro|river'] = 0.001*ct*gen_des(time(n.generators_t.p.filter(like ='ror').filter(like =country).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|Electricity|Hydro|dam'] = 0.001*ct*gen_des(time(n.storage_units_t.p.filter(like ='hydro').filter(like =country).sum(axis=1)))
+              var[sh][sheet_var[sheet]+'|Electricity|Hydro'] = var[sh][sheet_var[sheet]+'|Electricity|Hydro|river']+var[sh][sheet_var[sheet]+'|Electricity|Hydro|dam']
+           
+              #Energy generation : storage (PHS, battery, H2 storage)
+
+              var[sh][sheet_var[sheet]+'|Flexibility|Electricity Storage|Medium duration'] = 0.001*ct*(gen_des(time(n.storage_units_t.p.filter(like ='PHS').filter(like =country).sum(axis=1)))+\
+               gen_des(time(n.stores_t.p.filter(like ='H2').filter(like =country).sum(axis=1))))
+            
+              var[sh][sheet_var[sheet]+'|Flexibility|Electricity Storage|Short duration'] = 0.001*ct*(gen_des(time(n.stores_t.p.filter(like ='battery').filter(like =country).sum(axis=1)))+\
+               gen_des(time(n.stores_t.p.filter(like ='Li ion').filter(like =country).sum(axis=1))))
+            
+              var[sh][sheet_var[sheet]+'|Flexibility|Electricity Storage'] =  (var[sh][sheet_var[sheet]+'|Flexibility|Electricity Storage|Medium duration']
+               +var[sh][sheet_var[sheet]+'|Flexibility|Electricity Storage|Short duration'])
+            
+              #Energy generation : Interconnect 
+
+              var[sh][sheet_var[sheet]+'|Flexibility|Interconnect Importing Capacity'] = 0.001*ct*(gen_des(time(abs(n.lines_t.p1[[i for i in n.lines.index if country in n.lines.bus0[i] or country in n.lines.bus1[i]]]).sum(axis=1)))\
```

Comment:
check double-counting of power flows in links

Reply:
It's fine for each country, but gets added twice for the whole Europe. I added a loop at the end of the main loop to fix this issue.